<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuddyWidget</class>
 <widget class="QWidget" name="BuddyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>47</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="lblUri">
     <property name="text">
      <string>test@127.0.0.1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string>offline</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSendIM">
     <property name="text">
      <string>IM</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnUpdate">
     <property name="text">
      <string>update</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnClear">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
